data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/consumer/impl/BaseConsumerVerticle.java

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Why this? How the unordered implementation would differ from the ordered one?

Reply:
Because the unordered implementation would have the "fsm" in the consumer verticle implementation, it won't just invoke `consumer.handler`. In any case, it's still subject to changes, take it as a checkpoint to fix the #589

Reply:
Sound good!